README.md

```diff
@@ -303,10 +303,9 @@ The benchmarks used to test the performance and parsing capabilities of NMODL Fr
 * [NMODL Benchmark](https://github.com/BlueBrain/nmodlbench)
 * [NMODL Database](https://github.com/BlueBrain/nmodldb)
 
-### Authors
 
-See [contributors](https://github.com/BlueBrain/nmodl/graphs/contributors).
+## Funding & Acknowledgment
 
-### Funding
+The development of this software was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government's ETH Board of the Swiss Federal Institutes of Technology. In addition, the development was supported by funding from the National Institutes of Health (NIH) under the Grant Number R01NS11613 (Yale University) and the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant Agreement No. 785907 (Human Brain Project SGA2).
```

Comment:
I have based the funding information on what was written before and the guidelines available internally in our Confluence pages.

I kindly ask @pramodk and @jamesgking to carefully review this. The guidelines recommends contacting someone from BBP for further clarification.

Reply:
This looks fine to me. It was how written in the manuscript as well.